### DISCUSSION TITLE
Logging functions not found?

### DISCUSSION THREAD

**Discussion by shiide** [jac3km4/redscript#97] [Q&A] [answered]
Might be a silly oversight on my part. I'm trying to log something and getting compilation errors for any Log*() calls saying the given functions are not found.

Edit: I'm on redscript 0.5.16 and game 2.0.

Can you give me a hint on what I'm missing?

**Comment by jac3km4** [comment, marked as answer]
The definitions of the logging functions were removed from the game. Their native counterparts still exist though and can be defined and used in redscript:
```
native func LogChannel(channel: CName, text: script_ref<String>);
```

**Reply by shiide** [reply]
Thank you, appreciated!
